### PR TITLE
SDK validation functions

### PR DESCRIPTION
**Pull request by dougollerenshaw**

### CODE REVIEW

**Comment by dougollerenshaw** [comment]
This PR contains code to perform SDK validation and generatee the associated validation plot. For example: https://dougollerenshaw.github.io/figures_to_share/sdk_validation_matrix.html

File: visual_behavior/validation/sdk.py [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+def is_ophys(behavior_session_id):
+    cache = get_cache()
+
+    behavior_session_table = cache.get_behavior_session_table()
+
+    return pd.notnull(behavior_session_table.loc[behavior_session_id]['ophys_session_id'])
+
+
+def bsid_to_oeid(behavior_session_id):
```
Comment (matchings) [inline visual_behavior/validation/sdk.py:30]: is there a comparable function in data_access.utilities? If not, this type of thing should live there

File: visual_behavior/validation/sdk.py [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+        where behavior_sessions.id = {}
+        '''.format(behavior_session_id)
+    )
+    if isinstance(oeid, pd.DataFrame):
+        return oeid.iloc[0][0]
+    else:
+        return oeid
+
+
+def get_sdk_session(behavior_session_id, is_ophys):
```
Comment (matchings) [inline visual_behavior/validation/sdk.py:45]: is there a reason not to have this in data_access and then just call it here?

File: visual_behavior/data_access/utilities.py [hunk 3]
```diff
@@ -1,4 +1,202 @@
...
+def model_outputs_available_for_behavior_session(behavior_session_id):
+    """
+    Check whether behavior model outputs are available in the default directory
+
+    :param behavior_session_id: 9-digit behavior session ID
+    :return: Boolean, True if outputs are available, False if not
+    """
+    model_output_dir = loading.get_behavior_model_outputs_dir()
+    model_output_file = [file for file in os.listdir(model_output_dir) if str(behavior_session_id) in file]
+    if len(model_output_file) > 0:
```
Comment (alexpiet) [inline visual_behavior/data_access/utilities.py:199]: The logic block starting at 199 should be rewritten to be one statement: `return len(model_output_file) > 0`

(one of my pet-peeves)

File: visual_behavior/validation/sdk.py (hunk 1 above)
Comment (dougollerenshaw) [reply visual_behavior/validation/sdk.py:30]: I moved this function to data_access.utilities.

File: visual_behavior/validation/sdk.py (hunk 2 above)
Comment (dougollerenshaw) [reply visual_behavior/validation/sdk.py:45]: Moved to data_access.utilities

File: visual_behavior/data_access/utilities.py (hunk 3 above)
Comment (dougollerenshaw) [reply visual_behavior/data_access/utilities.py]: fixed. thanks for the suggestion.

**Pull request merged**